src/mono/mono/metadata/icall-def-netcore.h

```diff
@@ -1,3 +1,10 @@
+#if defined(TARGET_WASM)
```

Comment:
Can't we use a managed md5 implementation as well ?

Reply:
we could but there's no managed implementation in referencesource so we'd have to write/find one. since this is still being discussed I didn't want to spend time on it, just demonstrate that it works.

Reply:
We have an old mono implementation in mcs/class/corlib/System.Security.Cryptography/MD5CryptoServiceProvider.cs
which has been in use for 15 years.